appveyor.yml

```diff
@@ -0,0 +1,31 @@
+# Test against this version of Node.js
+environment:
+  matrix:
+    # node.js
+    - nodejs_version: "0.10"
+    - nodejs_version: "0.12"
+    # io.js
+    - nodejs_version: "1.0"
+
+# Install scripts. (runs after repo cloning)
+install:
+  # Get the latest stable version of Node.js or io.js
+  - ps: Install-Product node $env:nodejs_version
+  # Ensure the latest npm version
+  - npm install -g npm
```

Comment:
Does it really work on Windows/Appveyor? For me it was still picking up the included npm, not the installed one. Fixing that would most likely require messing with the `%PATH%` env variable.

I just settled to remove Node 0.10 from the setup in my packages; it's already tested via Travis and Node >=0.12 have fully updated npm@2 so they should work fine.